### ISSUE TITLE
no matches for kind "Certificate" in version "cert-manager.io/v1alpha2"

### ISSUE THREAD

**Issue by amoldavsky** [SeldonIO/seldon-core#4134]
looks like seldon-core-operator helm charts are not compatible with any cert-manager version above 1.1.1 (current is 1.8.0). Trying to run the Helm installation following the docs results in the following error:
```
Error: UPGRADE FAILED: [unable to recognize "": no matches for kind "Certificate" in version "cert-manager.io/v1alpha2", unable to recognize "": no matches for kind "Issuer" in version "cert-manager.io/v1alpha2", unable to recognize "": no matches for kind "ValidatingWebhookConfiguration" in version "admissionregistration.k8s.io/v1beta1"]
```

digging into the cert manager CRDs installed with 1.8.0 shows that indeed there is no `cert-manager.io/v1alpha2` version. There is only `v1`. 

Going through the releases on cert-manager's Github repo shows that they have removed these CRDs altogether after `v1.1.1` . This is the last these were in the repo:
https://github.com/cert-manager/cert-manager/blob/v1.1.1/deploy/crds/crd-certificaterequests.v1beta1.yaml

Basically, all referencing templates in `seldon-core-operator` need to be updated to `v1` or the docs should outline which versions of cert-manager are compatible with the particular versions of seldon-core.

This will probably not be an issue for people upgrading from older versions as the CRDs most likely still be there but I think new installs will be affected.

A temporary solution would be to instruct users to install the older cert-manager CRDs first and than the latest.

**Comment by cliveseldon** [comment]
This was fixed by https://github.com/SeldonIO/seldon-core/pull/3888

Which version of seldon-core are you running? It was available from 1.13.0

Closing this.

**Comment by amoldavsky** [comment]
I am running 1.7.0 and was trying to upgrade to 1.11.2
